colorcoder/Main.java

```diff
@@ -1,120 +1,8 @@
 package colorcoder;
 
-public class Main {
-    enum MajorColor {
-        WHITE(0),
-        RED(1),
-        BLACK(2),
-        YELLOW(3),
-        VIOLET(4);
-        private int index;
-        private MajorColor(int index) {
```

Comment:
assuming enums are used here to retrict the creation of other Major or minor colors